static/app/views/monitors/components/overviewTimeline/types.tsx

```diff
@@ -16,3 +18,9 @@ export interface TimeWindowOptions {
 }
 
 export type TimeWindowData = Record<TimeWindow, TimeWindowOptions>;
+
+export type MonitorBucketData = [timestamp: number, envData: MonitorBucketEnvMapping][];
+
+export type MonitorBucketEnvMapping = {
+  string: Record<CheckInStatus, number>;
+};
```

Comment:
is this slightly wrong? you want

```suggestion
 type MonitorBucketEnvMapping = Record<string, Record<CheckInStatus, number>>;
```

Reply:
aahhh yes missed this